TruexGoogleReferenceApp/PlayerViewController.swift

```diff
@@ -63,9 +63,21 @@ class PlayerViewController: UIViewController,
         streamManager.delegate = self
         playerViewController.delegate = self
 
+        listenForApplicationEvents()
         /* [OPTIONAL] */ listenForPlayerStatus()
     }
     
+    private func listenForApplicationEvents() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(self.applicationDidEnterBackground),
+                                               name: UIApplication.didEnterBackgroundNotification,
+                                               object: nil)
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(self.applicationWillEnterForeground),
+                                               name: UIApplication.willEnterForegroundNotification,
```

Comment:
Is there a better notification we can use? This one sounds like the app will enter the foreground, but hasn't quite yet. Just asking -- I don't know what notifications are available to us.

Reply:
That's a great point.

I think UIApplication.didBecomeActiveNotification would be a better fit for the resume callback.

Reply:
https://developer.apple.com/documentation/uikit/uiapplicationdidbecomeactivenotification